thuyetnv/thuyetnv/index.php

```diff
@@ -0,0 +1,22 @@
+<?php  session_start();
+include "config.php";
+include "model/model.php";
+
+$controller = isset($_GET["controller"]) ? $_GET["controller"]:"";
+
+if ($controller == "login")
```

Comment:
Toàn bộ đoạn code này viết khá rối, không theo chuẩn chung. Tại sao lại kiểm tra riêng với controller login và logout, còn lại set thành controller home, rồi sau đó lại kiểm tra để overwrite bằng controller đúng theo giá trị?